recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "4.1.1" %}
+{% set version = "5.1.2" %}
 
 package:
   name: ipywidgets
   version: {{ version }}
 
 source:
   fn: ipywidgets-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/i/ipywidgets/ipywidgets-{{ version }}.tar.gz
-  md5: 491affdef8e0586eeeb299226ff634aa
+  url: https://github.com/ipython/ipywidgets/archive/{{ version }}.tar.gz
+  sha256: bdd8a4e6736e1240ce20a975f838c1f9a5cd044df733657d354af53267ab21df
 
 build:
   number: 0
-  script: python setup.py install
+  script: python -m pip install --no-deps .
```

Comment:
- pip writes correct metadata on installation, which distutils does not
- if setuptools is ever imported unconditionally, then `setup.py install` should *never* be used, so it's protection against that eventuality as well
-  There's no downside to using pip here that I'm aware of (that doesn't mean there isn't one)